Sources/BraintreePayPalMessaging/BTPayPalMessagingPageType.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+import PayPalMessages
+
+/// Message location within an application
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTPayPalMessagingPageType {
+
+    /// Home view
+    case home
+
+    /// Individual product details view
+    case productDetails
+
+    /// Shopping cart view
+    case cart
+
+    /// Popover shopping cart view that covers part of the view
+    case miniCart
+
+    /// Checkout view
+    case checkout
+
+    /// Search results
+    case searchResults
+
+    var pageType: PayPalMessagePageType {
```

Comment:
update to align with the rest of the module:
```suggestion
    var pageTypeRawValue: PayPalMessagePageType {
```

Reply:
Updated: 0a040ea3ddb423acb988f33678ba8c0aaf7dc18c